site/content/kapp-controller/docs/develop/authoring-command.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Authoring Command Reference
+---
+
+## Package
+
+`kctrl` authoring commands help users generate resources that interact with `kapp-controllers` packaging layer.
+
+### Initialising the package
+The `package init` command takes some user inputs and creates boilerplate for package creation.  It should be run before using the `release` command. 
+```bash
+$ kctrl package init
+```
+Supported flags:
+- `--chdir` _string_, Working directory with package-build and other config
+
+Note: We always suggest to run `pkg init` in an interactive flow.
+ 
+### Releasing the Package
+The `package release` command is used to generate Package and PackageMetadata resources.
+```bash
+$ kctrl package release -v v1.0.0 --repo-output packages
+```
+Supported flags:
+- `-v`, `--version` _string_, Version to be released
+- `--repo-output` _string_, Output location for artifacts in repository bundle format
+- `--copy-to` _string_, Output location for artifacts (default "carvel-artifacts")
+- `--chdir` _string_, Working directory with package-build and other config
+
+## Package Repository
+### Releasing a Package Repository
+The `package repository release` command publishes a PackageRepository using the output of `--repo-output` flag from the `package release` command.
+```bash
+$ kctrl package repository release 
```

Comment:
nit: We dont give -v flag here but giving it in `pkg release`. I feel there is inconsistency.

Reply:
Yup make sense added it.